lib/uploadcare/uploadcare_settings.rb

```diff
@@ -4,4 +4,5 @@ module Uploadcare
   PUBLIC_KEY = ENV.fetch('UPLOADCARE_PUBLIC_KEY')
   SECRET_KEY = ENV.fetch('UPLOADCARE_SECRET_KEY')
   AUTH_TYPE = 'Uploadcare.Simple'
+  MULTIPART_SIZE_THRESHOLD = 10 * 1024 * 1024
```

Comment:
I believe that this should be 100 MB, not 10 MB by default.
```suggestion
  MULTIPART_SIZE_THRESHOLD = 100 * 1024 * 1024
```

Reply:
10 MB is minumum size, 100 is maximum for regular uploads

https://uploadcare.com/docs/api_reference/upload/request_based/

Reply:
minimum does not mean that we should use it.
please set it to 100 MB with a way to override